includes/class-wc-payments-features.php

```diff
@@ -351,7 +351,7 @@ public static function should_use_stripe_billing() {
 	 * @return bool
 	 */
 	public static function is_stripe_ece_enabled(): bool {
-		return '1' === get_option( self::STRIPE_ECE_FLAG_NAME, '0' );
+		return (bool) get_option( self::STRIPE_ECE_FLAG_NAME, '1' );
```

Comment:
I like the simplicity of this change to using (bool), but I think it would be better to stick to the pattern in the rest of the file for consistency. And while not likely to ever happen, setting `_wcpay_feature_stripe_ece` to "off" will still have ECE enabled.

Reply:
I refactored it in https://github.com/Automattic/woocommerce-payments/pull/9118/commits/7b263ef1b0085ae970e18bc5955eb7f212134865 for consistency. 
However, this approach won’t handle cases where the values `on` or `off` are used, but this limitation applies to all our feature flags.